src/Uno.UI/UI/Xaml/Controls/Grid/Grid.Layout.cs

```diff
@@ -1099,6 +1102,28 @@ private Point GetChildrenOffset()
 			return MeasureElement;
 		}
 
+		internal double GetActualWidth(ColumnDefinition columnDefinition)
```

Comment:
When discussion the feature, I though that ActualWidth/Height were DPs, but are in fact standard properties. Why did you choose to find the values back to the grid over of storing the result in the ColumnDefinition ?